src/Input/Select/Select.test.tsx

```diff
@@ -211,6 +219,12 @@ describe('when the enter key is pressed on an option', () => {
     const { selectList } = setupSelectOptionFromMenuWithKeyHandling();
     expect(selectList.hasAttribute('open')).toBe(false);
   });
+
+  it('should float the label ', () => {
+    const { selectLabel } = setupSelectOptionFromMenuWithKeyHandling();
+    const style = window.getComputedStyle(selectLabel);
+    expect(style.transform).not.toEqual('');
```

Comment:
I think we can use `toHaveStyle` instead.
```
expect(selectLabel).toHaveStyle('transform: translate3d(-15px,-20px,0);');
```

Reply:
Thanks for the tip! Implemented this :)